packages/lab/stories/overlay/overlay.stories.tsx

```diff
@@ -4,78 +4,108 @@ import {
   CheckboxGroup,
   StackLayout,
   Tooltip,
+  useId,
 } from "@salt-ds/core";
-import React, { ChangeEvent } from "react";
+import React, { ChangeEvent, useState } from "react";
 import { StoryFn, Meta } from "@storybook/react";
 
 import {
   Overlay,
   OverlayPanel,
   OverlayProps,
   OverlayTrigger,
+  OverlayPanelCloseButton,
+  OverlayPanelContent,
 } from "@salt-ds/lab";
 
 import "./overlay.stories.css";
 
 export default {
   title: "Lab/Overlay",
-  component: Overlay,
```

Comment:
Removed because this was always passing through an empty `onOpenChange` function in {..args } which was messing up the stories